### DISCUSSION TITLE
Interested in contribution for GSoc

### DISCUSSION THREAD

**Discussion by Irene-123** [SeaQL/summer-of-code#14] [General]
Hello developers! I'm Kirti, a student and intern @HPE. I'm learning rust and want to make some Open Source contributions for GSoC and came across this project. I went through some [issues](https://github.com/SeaQL/seaography/issues) of seaography and was wondering if I could make a start by contributing any small task, Let me know, Thank you so much!

**Comment by billy1624** [comment]
Hey @Irene-123, welcome to SeaQL! Any contributions are welcomed! Please start with the one that interested you, propose a implementation plan to the issue then draft a PR based on that. Let me know if you need help along the way :)

**Comment by Irene-123** [comment]
Is there a documentation or guide for building the [seaography](https://github.com/SeaQL/seaography) project on one's system?

**Reply by billy1624** [reply]
You simply run `cargo b` to build the project! I'd suggest you to follow the getting started to setup an example application on your local machine. https://github.com/SeaQL/seaography#quick-start---ready-to-serve-in-3-minutes
